datastore/uncached.go

```diff
@@ -137,3 +144,47 @@ func (s *DataStore) GetAllConfigMaps() (runtime.Object, error) {
 func (s *DataStore) GetAllVolumeAttachments() (runtime.Object, error) {
 	return s.kubeClient.StorageV1().VolumeAttachments().List(context.TODO(), metav1.ListOptions{})
 }
+
+// FindARandomReadyNodeForVolumeAttachment return a random ready node to attach
+// the volume.
+// Return error if there is no ready node
+func FindARandomReadyNodeForVolumeAttachment(v *longhornclient.Volume, api *longhornclient.RancherClient) (string, error) {
```

Comment:
It's better not to use `longhornclient` in this package. You can get the node list before invoking this function then make the node list as an input.

Reply:
Fixed.

Reply:
I mean, it's better not to use `"github.com/longhorn/longhorn-manager/client"` in ** the whole package** `datastore` rather than this go file.

Reply:
If it's hard to modify this function without introducing `"github.com/longhorn/longhorn-manager/client"`, you can directly add a similar function in csi rather than moving this function here.

Reply:
Thanks, addressed.